tests/sentry/relay/test_config.py

```diff
@@ -120,20 +129,50 @@ def test_get_experimental_config(mock_sentry_sdk, _, default_project):
     assert mock_sentry_sdk.capture_exception.call_args == mock.call(SOME_EXCEPTION)
 
 
+@pytest.mark.django_db
+@mock.patch("sentry.relay.config.capture_exception")
+def test_get_experimental_config_transaction_metrics_exception(
+    mock_capture_exception, default_project
+):
+    keys = ProjectKey.objects.filter(project=default_project)
+    default_project.update_option("sentry:breakdowns", {"invalid_breakdowns": "test"})
+    # wrong type
+    default_project.update_option("sentry:transaction_metrics_custom_tags", 42)
+
+    with Feature({"organizations:transaction-metrics-extraction": True}):
+        cfg = get_project_config(default_project, full_config=True, project_keys=keys)
+
+    # we check that due to exception we don't add `d:transactions/breakdowns.span_ops.ops.{op_name}@millisecond`
+    assert (
+        "breakdowns.span_ops.ops"
+        not in cfg.to_dict()["config"]["transactionMetrics"]["extractMetrics"]
+    )
+    assert cfg.to_dict()["config"]["transactionMetrics"]["extractCustomTags"] == []
```

Comment:
What is this assertion trying to verify? If the goal is to ensure `config.transactionMetrics.extractCustomTags` exists, we should validate it with `is not None` instead of `== []`?

Reply:
this assert validates that we raise exception when adding `extractCustomTags` and we fallback to empty list.
So won't work:
```
In [1]: [] is None
Out[1]: False
```

Reply:
I see. Perfect then!